src/components/WorkshopsForm/WorkshopFormModalContent.jsx

```diff
@@ -47,18 +49,32 @@ const ModalContent = () => {
       setModalContent(
         <div className={css.workshopsSuccessModalContainer}>
           <h2 className={css.workshopsSuccessModalHeader}>
-            Formularz zapisu na warsztaty został wysłany! Dziękujemy za aktywne
-            dołączenie do wydarzenia Uwaga! Smartfon.
+            {t('workshopsForm.titleModal')}
           </h2>
           <p className={css.workshopsSuccessModalParagraph}>
-            Zapraszamy do zapoznania się z programem konferencji.
+            {t('workshopsForm.textModal1')}
           </p>
-          <button>Zobacz program</button>
+          <Button
+            type="button"
+            variant="secondary"
+            content={t('workshopsForm.buttonModal1')}
+            onClick={() => {
+              scrollToAnchor('program')
+              setIsModalVisible(false)
```

Comment:
tutaj zamiast tego setModalVisible musisz dać closeModal() bo w tej chwili jak klikniesz na buttona jednego albo drugiego to nie działa scroll strony